From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Mon, 20 Mar 2023 17:48:40 -0600
Subject: [PATCH] kernelbase: HACK: Add --use-angle=d3d9 command line option
 for nw.exe.exe.

CW-Bug-Id: #22042
Valve-commit: f088e529cb61763c527d5cd33455f0f53eba7721
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-wine-proton-rebase.md
Offered-upstream: no
---
 dlls/kernelbase/process.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/kernelbase/process.c b/dlls/kernelbase/process.c
index 13d9f42..10719cd 100644
--- a/dlls/kernelbase/process.c
+++ b/dlls/kernelbase/process.c
@@ -514,6 +514,7 @@ static const WCHAR *hack_append_command_line( const WCHAR *cmd )
         {L"Paradox Launcher.exe", L" --use-gl=swiftshader --in-process-gpu"},
         {L"Montaro\\nw.exe", L" --use-gl=swiftshader"},
         {L"OlympiaRising.exe", L" --use-gl=swiftshader"},
+        {L"nw.exe.exe", L" --use-angle=d3d9"},
     };
     unsigned int i;
 
